package.json

```diff
@@ -1,12 +1,11 @@
 {
   "name": "vtex",
-  "version": "2.106.2",
+  "version": "2.106.3-beta.3",
   "description": "The platform for e-commerce apps",
   "bin": "bin/run",
   "main": "build/api/index.js",
   "scripts": {
-    "exe": "node scripts/make-executable.js",
-    "watch": "bash ./scripts/symlink.sh && yarn nodemon && yarn oclif-dev manifest",
+    "watch": "yarn build-clean && bash ./scripts/symlink.sh && yarn nodemon",
```

Comment:
Why you removed `oclif-dev manifest` ?

Reply:
Once we execute nodemon here the nodemon process wil run undefinately until the user cancel it, so `oclif-dev manifest` will not be run here.
The nodemon process will watch the repository files for changes and run `build-incremental` which will run `oclif-dev manifest` for each file change that happens:
https://github.com/vtex/toolbelt/blob/3afe9a0a8bb892d0769be3c29060995c92242b2a/nodemon.json#L4